OutOfSchool/OutOfSchool.WebApi/Services/Communication/CommunicationService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+using Newtonsoft.Json;
+
+using OutOfSchool.Common;
+using OutOfSchool.WebApi.Config;
+using OutOfSchool.WebApi.Services.Communication.ICommunication;
+
+namespace OutOfSchool.WebApi.Services.Communication
+{
+    public class CommunicationService : ICommunicationService
```

Comment:
it will probably be good to move the internal communication between our services to GRPC, because what we're doing is essentially remote procedure calls. (p.s. not now but some time later :), maybe just add a TODO)